suite-native/feature-flags/src/featureFlagsSlice.ts

```diff
@@ -1,33 +1,41 @@
 import { Platform } from 'react-native';
 
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
-export interface FeatureFlagsState {
-    isDeviceConnectEnabled: boolean;
+export enum FeatureFlag {
+    IsDeviceConnectEnabled = 'isDeviceConnectEnabled',
+    IsPassphraseEnabled = 'isPassphraseEnabled',
 }
 
+export type FeatureFlagsState = Record<FeatureFlag, boolean>;
+
 export type FeatureFlagsRootState = {
     featureFlags: FeatureFlagsState;
 };
 
 export const featureFlagsInitialState: FeatureFlagsState = {
-    isDeviceConnectEnabled: Platform.OS === 'android',
+    [FeatureFlag.IsDeviceConnectEnabled]: Platform.OS === 'android',
+    [FeatureFlag.IsPassphraseEnabled]: Platform.OS === 'android',
```

Comment:
initial value should be false or true only in development mode, right?

Reply:
Yes, I will probably give this `false` altogether for now and when it makes sense, I will turn it on for development and android, what do you think? Until there is some useful flow, I'd rather have it on demand so unnecessary bugs are not reported. https://github.com/trezor/trezor-suite/pull/10693/commits/75eea5032a29fd71b878a2f0562a4a0566a5e18c

Reply:
makes sense 👌